### DISCUSSION TITLE
zopen init, alternative site to get cacert.pem?

### DISCUSSION THREAD

**Discussion by fswarbrick** [zopencommunity/meta#752] [Q&A] [answered]
My shop blocks most non-U.S. sites, so I am unable to connect to both curl.se and curl.haxx.se.  Is there an alternative site I can download the cacert.pem file from?  Or a way to bypass this step?

**Comment by IgorTodorovskiIBM** [comment, marked as answer]
Hi, it's on our meta repo - https://github.com/ZOSOpenTools/meta/blob/e5ef78f8228b05bfbd046ace0a693c3d568fcf0e/cacert.pem

@DevonianTeuchter - do you recall if there's a way to bypass the cacert.pem download? Perhaps we should add an option to avoid downloading it or print a warning rather than a hard error.

**Reply by fswarbrick** [reply]
That works.  Thanks.

Can you add it as a second alternative location for zopen to try?
